.changeset/grumpy-poems-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
```

Comment:
Let's add the `console` and `myaccount` as well.

Reply:
`console` will be enough. As it's console specific change.

Reply:
We need to bump both `console` and `myaccount` to maintain the consistency of versioning.

Reply:
resolved in https://github.com/wso2/identity-apps/pull/4727/commits/b533228182a5092b13584912e7032c1147eacc69